server/src/main/java/org/opensearch/common/settings/ClusterSettings.java

```diff
@@ -723,7 +722,10 @@ public void apply(Settings value, Settings current, Settings previous) {
 
                 // Concurrent segment search settings
                 SearchService.CLUSTER_CONCURRENT_SEGMENT_SEARCH_SETTING,
-                SearchService.CONCURRENT_SEGMENT_SEARCH_TARGET_MAX_SLICE_COUNT_SETTING
+                SearchService.CONCURRENT_SEGMENT_SEARCH_TARGET_MAX_SLICE_COUNT_SETTING,
+
+                RemoteStoreSettings.CLUSTER_REMOTE_INDEX_SEGMENT_METADATA_RETENTION_MAX_COUNT_SETTING,
+                RemoteStoreSettings.CLUSTER_REMOTE_TRANSLOG_BUFFER_INTERVAL_SETTING
```

Comment:
We may also include settings in `RemoteStoreNodeService` and `RemoteStorePressureSettings` as inner class of `RemoteStoreSettings` but can be done later to keep the scope limited for this PR.

Reply:
Lets create an issue, this can be a good first issue as well.

Reply:
Created: https://github.com/opensearch-project/OpenSearch/issues/12955